nginx.conf

```diff
@@ -20,7 +20,7 @@ server {
     server_name _;
 
     # Redirects
-    rewrite ^.* https://events.canonical.com/e/summit2022 redirect;
```

Comment:
do we need to also keep this redirect?

Reply:
As far as I understand, no. This is meant to replace that redirect altogether